### PR TITLE
OSDOCS-9339

### PR DESCRIPTION
**Pull request by smunje1**
[OSDOCS-9339](https://issues.redhat.com/browse/OSDOCS-9339): [rosa-] Issue in file upgrading/rosa-upgrading.adoc

Aligned team: Service Delivery
OCP version for cherry-picking: enterprise-4.14+
JIRA issues: [OSDOCS-9339](https://issues.redhat.com/browse/OSDOCS-9339)
Preview pages: [ROSA build preview](https://70243--ocpdocs-pr.netlify.app/openshift-rosa/latest/upgrading/rosa-upgrading#rosa-upgrade-ocm_rosa-upgrading)
SME review **completed**: @lalan7
QE review **completed**:  @jmekkatt 
Peer review **completed**: @mletalie 

Note: We have SME approval in [Jira comment](https://issues.redhat.com/browse/OSDOCS-9339?focusedId=23834538&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-23834538).

### CODE REVIEW

File: modules/rosa-upgrading-manual-ocm.adoc [hunk 1]
```diff
@@ -33,8 +33,14 @@ endif::sts[]
 . If you select an update version that requires approval, provide an administrator’s acknowledgment and click *Approve and continue*.
 +
 For information about administrator acknowledgment, see xref:./../upgrading/rosa-upgrading-cluster-prepare.adoc#upgrade-49-acknowledgement_rosa-updating-cluster-prepare[Administrator acknowledgment when upgrading to OpenShift 4.9].
-. In the *Node draining* pane, select a grace period interval from the list. The grace period enables the nodes to gracefully drain before forcing the pod eviction. The default is *1 hour*.
+. In the *Node draining* pane, select a grace period interval from the list. The grace period enables the nodes to gracefully drain before forcing the pod eviction. The default is *3 hours*, and click *Save*.
```
Comment (jmekkatt) [inline modules/rosa-upgrading-manual-ocm.adoc:36]: The default grace period value is set to 1 hour from OCM UI ROSA wizard. 
image
New change refers to 3 hours. Could you please let me know details of the same?

Comment (jmekkatt) [reply modules/rosa-upgrading-manual-ocm.adoc:36]: Also see the Setting tab of a ready ROSA cluster as below
image
@smunje1 Kindly have a look. Thanks

Comment (smunje1) [reply modules/rosa-upgrading-manual-ocm.adoc:36]: Hello @lalan7 . Could you please help with your input here? Thanks!

Hi @jmekkatt . I have made the updates as per the Jira description and discussion in [Jira comment](https://issues.redhat.com/browse/OSDOCS-9339?focusedId=23778448&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-23778448). Thanks!

Comment (lalan7) [reply modules/rosa-upgrading-manual-ocm.adoc:36]: @smunje @jmekkatt From what I understand here the default period is not what need to be changed here. In the Jira the 3 hours was on the reproducer has a example.

Comment (smunje1) [reply modules/rosa-upgrading-manual-ocm.adoc:36]: @lalan7 @jmekkatt Based on above discussion, I am revoking the change made on Line 36.

File: modules/rosa-upgrading-manual-ocm.adoc [hunk 2]
```diff
@@ -35,6 +35,12 @@ endif::sts[]
 For information about administrator acknowledgment, see xref:./../upgrading/rosa-upgrading-cluster-prepare.adoc#upgrade-49-acknowledgement_rosa-updating-cluster-prepare[Administrator acknowledgment when upgrading to OpenShift 4.9].
 . In the *Node draining* pane, select a grace period interval from the list. The grace period enables the nodes to gracefully drain before forcing the pod eviction. The default is *1 hour*.
 . In the *Update strategy* pane, click *Save* to apply your update strategy.
++
```
Comment (jmekkatt) [inline modules/rosa-upgrading-manual-ocm.adoc:38]: @smunje1 Hi, The new note would be more catchy if we could move it below the node draining step(after line 36). WDYT?
Other sections LGTM. Thanks

Comment (smunje1) [reply modules/rosa-upgrading-manual-ocm.adoc:38]: Yes. That's a great suggestion. Thanks!

**Comment by jmekkatt** [comment]
Changes looks fine for me.

**Review by mletalie — COMMENTED** [review]
/label peer-review-in-progress

**Review by mletalie — COMMENTED** [review]
/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label-peer-review-done
/lgtm

**Review by mletalie — COMMENTED** [review]
/label peer-review-done

**Comment by stevsmit** [comment]
LGTM

**Pull request merged**